### PR TITLE
feat - new api to query transaction message

### PR DESCRIPTION
**Pull request by ryukato**
### What kind of change does this PR introduce? 
* [ ] bug fix
* [x] feature
* [ ] docs
* [ ] update

### What is the new behavior?
New api to query transaction messages

### CODE REVIEW

**Review by dduck00 — DISMISSED** [review]

File: lib/response.ts [hunk 1]
```diff
@@ -383,3 +383,19 @@ export class IssueProxyResponse {
...
+    readonly requestType: string,
+    readonly details: any
+  ) { }
+}
+
+export class TxMessageListResponse {
+  constructor(
+    readonly messages: Array<TxMessageWithDetailResponse>,
+    readonly prePageToken?: string,
+    readonly nextPageToken?: string,
```
Comment (yeojin-dev) [inline lib/response.ts]: prePageToken and nextPageToken fields seem to be returning empty strings instead of null.

**Review by yeojin-dev — DISMISSED** [review]

File: lib/response.ts [hunk 2]
```diff
@@ -383,3 +383,19 @@ export class IssueProxyResponse {
...
+    readonly requestType: string,
+    readonly details: any
+  ) { }
+}
+
+export class TxMessageListResponse {
+  constructor(
+    readonly messages: Array<TxMessageWithDetailResponse>,
+    readonly prePageToken: string = "",
+    readonly nextPageToken: string= "",
```
Comment (xymz) [inline lib/response.ts]: chore: could you do align the code style?

File: lib/response.ts [hunk 3]
```diff
@@ -383,3 +383,19 @@ export class IssueProxyResponse {
...
+  ) { }
+}
```
Comment (xymz) [inline lib/response.ts]: chore: I think should be newline here

**Review by xymz — DISMISSED** [review]

File: lib/response.ts (hunk 2 above)
Comment (ryukato) [reply lib/response.ts]: ?? please let me know in more details.

Comment (ryukato) [reply lib/response.ts]: Fixed at aedf661ed352d69f35e7f4c43536a731c9b7ea6b

**Review by junbae2 — APPROVED** [review]

**Review by junbae2 — APPROVED** [review]

**Review by junbae2 — APPROVED** [review]

**Review by yeojin-dev — APPROVED** [review]

**Pull request merged**